### ISSUE TITLE
Clear "Share With Time Code" When Closing Video 

### ISSUE THREAD

**Issue by akano12** [libre-tube/LibreTube#4830]
### Describe your suggested feature

Clear the share with time code option after closing a video. 

### Other details

LT currently remember the share with time code option across videos, set the toggle to false after copying link or closing the video 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
In my opinion, this should stay as it is, since if a user likes sharing videos with timestamps, why shouldn't we make there life easier by remembering that decision?

**Comment by akano12** [comment]
> In my opinion, this should stay as it is, since if a user likes sharing videos with timestamps, why shouldn't we make there life easier by remembering that decision?

Isn't it better to rework it to simply copy the link with timestamp when tapping on share with timecode instead of the current way of sharing?

**Comment by Bnyro** [comment]
No, in my opinion the user should be able to modify the timestamp.

**Comment by Bnyro** [comment]
Converting to a discussion.
